addon/components/pie-chart.js

```diff
@@ -227,12 +238,40 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
   numSlices: Ember.computed.alias('finishedData.length'),
 
   // Offset slices so that the largest slice finishes at 12 o'clock
-  startOffset: Ember.computed('finishedData', function() {
-    var data = this.get('finishedData');
-    var sum = data.reduce(function(p, d) {
+  //
+  // Rotate the pie and concentrate all small slices at 8 to 10 o'clock if
+  // there is a high density of small slices inside the pie.
+  startOffset: Ember.computed('finishedData', 'sortKey', function() {
+    var finishedData = this.get('finishedData');
+    var sum = finishedData.reduce(function(p, d) {
```

Comment:
Yep, though there are some edge cases when rounding forces the percentage sum to be close (but not quite) 100. For example, if I have data points with percentage values (1.3%, 1.3%, 1.4%, and 96%) when  maxDecimalPlace = 0 by default. These data points will round to (1%, 1%, 1%, and 96%) which sums to 99%.

Although I could change it to be 100, I'm somewhat hesitant because of this edge case. Either way, I think it "should" be OK. I'll add a comment here for now